apps/tangle-dapp/app/page.tsx

```diff
@@ -9,16 +9,30 @@ const AccountPage: FC = () => {
   return (
     <div className="flex flex-col gap-5">
       <div className="flex gap-6 flex-col xl:flex-row">
-        <AccountSummaryCard className="md:max-w-full xl:max-w-[556px]" />
+        <Suspense
+          fallback={
+            <SkeletonLoader className="rounded-2xl md:max-w-full xl:max-w-[556px] min-h-[274px]" />
+          }
+        >
+          <AccountSummaryCard className="md:max-w-full xl:max-w-[556px]" />
+        </Suspense>
 
-        <RecentTxContainer />
+        <Suspense
+          fallback={
+            <SkeletonLoader className="rounded-2xl min-h-[170px] xl:min-h-[274px]" />
+          }
+        >
+          <RecentTxContainer />
+        </Suspense>
       </div>
 
       <Typography variant="h4" fw="bold">
         Manage Balances
       </Typography>
 
-      <BalancesTableContainer />
+      <Suspense fallback={<SkeletonLoader className="rounded-2xl h-[190px]" />}>
+        <BalancesTableContainer />
+      </Suspense>
     </div>
```

Comment:
These components already utilize skeleton loaders underneath.. if we add this here we might have too many skeleton loaders? Hmm

Reply:
You can check the replay to see if the UI is still good or not. I find this still acceptable